react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -13,6 +13,7 @@ const actionsCreators = {
   login: userData => async dispatch => {
     const response = await usersService.userLogin(userData);
     if (response.ok) {
+      window.localStorage.setItem('token', response.data);
```

Comment:
You don't need access window explicitly. All your variables are saved in your window object by default, so JS will always try to find them there. 

```suggestion
      localStorage.setItem('token', response.data);
```